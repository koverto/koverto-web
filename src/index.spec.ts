describe("entrypoint", () => {
  it("renders the app container", () => {
    require("./index")
    const app = document.getElementById("app")

    expect(app).not.toBeNull()
    expect(app).toMatchSnapshot()
  })
})
