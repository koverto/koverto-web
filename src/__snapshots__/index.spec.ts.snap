// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`entrypoint renders the app container 1`] = `
<div
  id="app"
>
  <h1>
    Hello, React!
  </h1>
</div>
`;
